### DISCUSSION TITLE
Outreachy  internship applicant

### DISCUSSION THREAD

**Discussion by EjuluWilson** [ModECI/MDF#217] [General]
Hi there! I am Ejulu Wilson,  an Outreachy internship applicant for the upcoming intake. I have 4 years of python programming experience and 2 in machine learning and the use of Jupyter notebooks. I will be graduating in May with a BSc in Computer Engineering. 
My interests are in ML/AI, robotics and Data Science.  

Of the Outreachy projects, I found the MDF to be the most interesting and most aligned with my career goals.  It gives me the opportunity to learn a great deal, put to use and grow my machine learning knowledge and programming skills. 

It will be such a great opportunity to contribute to this amazing project.

**Comment by pgleeson** [comment]
Welcome @EjuluWilson, thanks for youyr interest in the project. Please take a look at #213 for some suggestions for initial tasks.
